### ISSUE TITLE
modal issue

### ISSUE THREAD

**Issue by jeffswitzer** [NathanaelA/nativescript-platform-css#2]
Great work as always Nathan!
I gave this plugin a shot and after adding this css
```
Button {
    background-color: red;
}
.android Button {
    background-color: purple;
}
```
The majority of the buttons show up as purple on my android device as expected. However when calling `alert("welcome");` the alert button shows up as red. Additionally all buttons nested in any modal pages show up as red on android.

**Comment by NathanaelA** [comment]
I'm not sure how easy the dialogs can be fixed; that is a very sticky issue, based on how everything works.        But the show modal, I would have thought should have worked out of the box.       I'll try and take a look into that one with the 3.0 revamping that has to happen.

**Comment by abhayastudios** [comment]
Hi Nathan, I am running into the same issue with modals as well. Any chance you'd get around to fixing this? Maybe now that modals have been reworked in NS 4.1 this should be easier to fix?

Thanks a lot!

**Comment by abhayastudios** [comment]
I see that the global variable nsPlatform from the nativescript-platform plugin is available inside the modal component. I am therefore guessing that the [Page events](https://github.com/NathanaelA/nativescript-platform-css/blob/master/src/platformcss.js#L93) that nativescript-platform-css is using is not fired inside a modal.

**Comment by abhayastudios** [comment]
Ok I was able to get it to work in modals by adding the `shownModallyFirst` event which is already supported by `nativescript-platform`. I created PR #18 for this.

**Comment by NathanaelA** [comment]
Thank you @abhayastudios for the fix - I have merged the PR; and published 1.6.6 which contains this.
